components/channel_header/channel_header.js

```diff
@@ -716,6 +727,7 @@ class ChannelHeader extends React.PureComponent {
                                 <span>
                                     {archivedIcon}
```

Comment:
Is it alright to show both the archived and shared icons here?

Reply:
Archive is shown at the beginning of the header and shared at the end, so they don't conflict:

![image](https://user-images.githubusercontent.com/223323/105865146-eb56f100-5ff2-11eb-91fe-cea46e8e9111.png)